<?xml version="1.0"?>
<launch>
  <!-- Запуск скрипта для исправления трансформаций -->
  <node pkg="basic_nav" type="fix_transforms.py" name="transform_fixer" output="screen">
    <param name="map_frame" value="map"/>
    <param name="odom_frame" value="odom"/>
    <param name="base_frame" value="base_link"/>
    <param name="publish_rate" value="50.0"/>
  </node>
  
  <!-- Установка начальной позиции -->
  <node pkg="basic_nav" type="set_initial_pose.py" name="set_initial_pose" output="screen">
    <param name="map_center_x" value="50.0"/>
    <param name="map_center_y" value="50.0"/>
    <param name="initial_yaw" value="0.0"/>
  </node>
</launch> 